### PR TITLE
[dv/doc] update dvsim in index.md

### PR DESCRIPTION
**Pull request by cindychip**
Follow PR #3443, this PR updates the run command to dvsim, and also RAL
auto-gen description for all existing IP's DV doc.

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

File: hw/ip/tlul/doc/dv_plan/index.md [hunk 1]
```diff
@@ -108,8 +108,7 @@ We are using our in-house developed [regression tool]({{< relref "hw/dv/tools/RE
 Please take a look at the link for detailed information on the usage, capabilities, features and known issues.
 Here's how to run a basic sanity test:
 ```console
-$ cd hw/ip/xbar/dv
-$ make TEST_NAME=xbar_sanity
+$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim_cfg.hjson -i xbar_main_sanity
```
Comment (cindychip) [inline hw/ip/tlul/doc/dv_plan/index.md:111]: @weicaiyang there are `xbar_main` and `xbar_peri`. I only add xbar_main here but i feel like maybe we need some description about how to run xbar_main and xbar_peri here.

**Review by sriyerg — APPROVED** [review]
LGTM! Please update template as well if required.

**Comment by cindychip** [comment]
Thanks @sriyerg  for the quick review. The template has already been updated in PR #3443

File: hw/ip/tlul/doc/dv_plan/index.md (hunk 1 above)
Comment (sriyerg) [reply hw/ip/tlul/doc/dv_plan/index.md:111]: I dont think we should add chip specific stuff to this file. Lets discuss more on this offline.

Comment (weicaiyang) [inline hw/ip/tlul/doc/dv_plan/index.md:111]: how about this?
```suggestion
$ $REPO_TOP/util/dvsim/dvsim.py $REPO_TOP/hw/top_earlgrey/ip/$XBAR_IP/dv/autogen/xbar_main_sim_cfg.hjson -i ${XBAR_IP}_sanity
// Note: $XBAR_IP could be xbar_main, xbar_peri, etc.
```

**Review by weicaiyang — APPROVED** [review]
thanks for updating this. LGTM

File: hw/ip/tlul/doc/dv_plan/index.md (hunk 1 above)
Comment (cindychip) [reply hw/ip/tlul/doc/dv_plan/index.md:111]: that works thanks Weicai for the suggestion. Let us discuss more with Sri if we should talk about anything top-level here. If so, will definitely use your suggestion.

Comment (sriyerg) [reply hw/ip/tlul/doc/dv_plan/index.md:111]: Also replace `top_earlgrey` with $CHIP as well and add a note indicating $CHIP = top_earlgrey, etc. This LGTM.

**Review by sriyerg — APPROVED** [review]

**Pull request merged**